src/main/java/org/folio/spring/data/OffsetRequest.java

```diff
@@ -38,6 +37,14 @@ public OffsetRequest(long offset, int limit, Sort sort) {
     this.sort = Objects.requireNonNull(sort, "Sorting cannot be null. Use Sort.unsorted() instead");
   }
 
+  public static OffsetRequest of(int offset, int limit) {
+    return of(offset, limit, Sort.unsorted());
```

Comment:
it's better to keep it in consistence with constructor: use DEFAULT_SORT

Reply:
Thanks! Fixed it.